components/admin/resources/questions-display.tsx

```
@@ -33,10 +33,10 @@ export function QuestionDisplay({
 }: QuestionDisplayProps) {
   const { toast } = useToast()
   const content = question.content as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)